appveyor.yml

```diff
@@ -110,12 +115,16 @@ for:
     - mingw32-make -j%JOBS% V=1
     - mingw32-make DESTDIR=../install install-nodoc
     - if not "%GEMS_FOR_TEST%" == "" ..\install\bin\gem install --no-document %GEMS_FOR_TEST%
+  before_test:
+    - cd %APPVEYOR_BUILD_FOLDER%
```

Comment:
Seems this line causes the next line failure.

Reply:
It seems like `before_test` was not appropriate for doing this either. I dropped `before_test` usage for msys2 https://github.com/ruby/ruby/pull/1998/commits/e618f1058ff89ce137cac23fa5341ade1852045c, and it seems to pass that part.

Reply:
@nobu and @k0kubun

I apologize for the stupid mistake.  I was 'retired' when the CI ran...

I had the 'cd' command in case testing crashed and left pwd in an unknown state.  Considering the other PR was running CLI bin stubs in the same folder as ruby.exe, I should have caught it...